docs/Load_Cell.md

```diff
@@ -0,0 +1,152 @@
+# Load Cells
+
+This document describes Klipper's support for load cells and load cell based
+probes.
+
```

Comment:
not knowing what a load cell does/means for a 3d printer, could you add a sentence or two describing it's usage/benefits for 3d printers/klipper?

Reply:
I made some updates to the documentation including a little bit more exposition at the top and a section about how to read the load cell gram force